install/mysql/plugin_formcreator_empty.sql

```diff
@@ -5,7 +5,7 @@ CREATE TABLE IF NOT EXISTS `glpi_plugin_formcreator_answers` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
   `plugin_formcreator_formanswers_id` int(11) NOT NULL,
   `plugin_formcreator_questions_id` int(11) NOT NULL,
-  `answer` text,
+  `answer` longtext,
```

Comment:
your base branch is probably late 

https://github.com/pluginsGLPI/formcreator/blob/develop/install/mysql/plugin_formcreator_empty.sql#L8